src/app/event/new-scheduled-event/new-scheduled-event.component.html

```diff
@@ -209,7 +209,31 @@
         Please wait...
       </span>
     </ng-container>
-    <ng-container *ngIf="!checkingEnvironments">
+    <ng-container *ngIf="noEnvironmentsAvailable">
+      <span>
+        No suitable environments found.
+      </span>
+    </ng-container>
+    <ng-container *ngIf="unavailableVMTs.length > 0">
+      <span>
+        No suitable environments for the following VMTs found:
```

Comment:
Can we change this to
"No suitable environments found for the following VM Templates:"
please?